Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -1770,4 +1753,47 @@ namespace Tensile
                       << " shiftPtrElemB=" << st.shiftPtrElemB << " depthUorMT0=" << st.depthUorMT0
                       << " depthUorMT1=" << st.depthUorMT1;
     }
+
+    std::ostream& operator<<(std::ostream& stream, const SizeMapping& sizeMapping)
+    {
+        return stream << std::right << std::setw(30)
+                      << "workGroupSize: " << sizeMapping.workGroupSize << std::endl
+                      << std::right << std::setw(30) << "threadTile: " << sizeMapping.threadTile
```

Comment:
Could you put std::right << std::setw(30) only once here?
It looks like all lines here are in the same single sentence.
We need to specify setw only once per sentence.

Reply:
This was intentional, I added these for better readability of the print (currently the output looks aligned on `:` and its more readable). Please let me know if you still want me to remove it. Thanks

Reply:
I would like to have comment from other reviewers.
Personally, I would like to make output for each parameter as a single line (like line 1766).
@babakpst, do you think the current one is more readable?

Reply:
After discussing with @babakpst, it looks like this is what we can do.
  setw : we cannot remove them. Need to specify one by one
  right: we need only once. We can remove remaining. Probably better to reset right setting after printing all parameters